File: app/src/main/java/com/forkmang/fragment/LocationScreen_Fragment.java
package com.forkmang.fragment;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.forkmang.R;
import com.forkmang.adapter.Wallet_Fragment_Adapter;
import com.forkmang.adapter.location_Fragment_Adapter;

public class LocationScreen_Fragment extends Fragment {

    RecyclerView recyclerView;
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.activity_location_screen, container, false);
        recyclerView = view.findViewById(R.id.wallet_recycleview);
        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));

        location_Fragment_Adapter location_fragment_adapter = new location_Fragment_Adapter(getActivity());
        recyclerView.setAdapter(location_fragment_adapter);

        return view;
    }

    /*@Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_location_screen);

        recyclerView = findViewById(R.id.wallet_recycleview);
        recyclerView.setLayoutManager(new LinearLayoutManager(LocationScreen.this));

        location_Fragment_Adapter location_fragment_adapter = new location_Fragment_Adapter(LocationScreen.this);
        recyclerView.setAdapter(location_fragment_adapter);
    }*/
}

File: app/src/main/java/com/forkmang/fragment/LocationScreen_Fragment.kt
package com.forkmang.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.forkmang.R
import com.forkmang.adapter.location_Fragment_Adapter

class LocationScreen_Fragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val view = inflater.inflate(R.layout.activity_location_screen, container, false)
        val recyclerView: RecyclerView = view.findViewById(R.id.wallet_recycleview)
        recyclerView.layoutManager = LinearLayoutManager(context)
        val location_fragment_adapter = location_Fragment_Adapter(requireActivity())
        recyclerView.adapter = location_fragment_adapter
        return view
    } /*@Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_location_screen);

        recyclerView = findViewById(R.id.wallet_recycleview);
        recyclerView.setLayoutManager(new LinearLayoutManager(LocationScreen.this));

        location_Fragment_Adapter location_fragment_adapter = new location_Fragment_Adapter(LocationScreen.this);
        recyclerView.setAdapter(location_fragment_adapter);
    }*/
}